src/extension/content-script/onendwebln.js

```diff
@@ -32,11 +32,15 @@ async function init() {
 
   injectScript(browser.runtime.getURL("js/inpageScript.bundle.js")); // registers the DOM event listeners and checks webln again (which is also loaded onstart
 
-  // extract LN data from websites
   browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
+    // extract LN data from websites
     if (request.action === "extractLightningData") {
       extractLightningData();
     }
+    // forward account selected messages to the website
```

Comment:
this comment is now wrong, it should be the inpage-script right? I am not sure if we can use a different value than "*", but I don't think it matters really, at least for this event.

Reply:
resolved